.github/workflows/tests.yml

```
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11-dev"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```
This should be enough now that actions/setup-python#414 has been merged.

---

Good idea, however it's been merged but not yet released:

https://github.com/hugovk/test/actions/runs/4371675366/jobs/7647785823

I'll ask on that PR about a release.